tests/mock_virtual_machine.h

```diff
@@ -37,6 +37,11 @@ struct MockVirtualMachine : public multipass::VirtualMachine
         ON_CALL(*this, management_ipv4()).WillByDefault(Return("0.0.0.0"));
         ON_CALL(*this, get_all_ipv4(_)).WillByDefault(Return(std::vector<std::string>{"192.168.2.123"}));
         ON_CALL(*this, ipv6()).WillByDefault(Return("::/0"));
+        ON_CALL(*this, ensure_vm_is_running()).WillByDefault(Invoke([]() { throw std::runtime_error("Not running"); }));
+    }
+
+    MockVirtualMachine() : MockVirtualMachine{""}
```

Comment:
Why add this?

Reply:
I added it to make the interface the same as the stub. But right, it's not needed. Removed.